packages/esm-patient-notes-app/src/notes/notes-detailed-summary.component.tsx

```diff
@@ -1,148 +1,171 @@
-import React, { useState, useEffect, Fragment } from 'react';
-import styles from './notes-detailed-summary.css';
-import capitalize from 'lodash-es/capitalize';
-import { EmptyState, SummaryCard } from '@openmrs/esm-patient-common-lib';
-import { Link } from 'react-router-dom';
+import React from 'react';
+import styles from './notes-overview.scss';
+import { EmptyState, ErrorState, launchStartVisitPrompt } from '@openmrs/esm-patient-common-lib';
 import { useTranslation } from 'react-i18next';
-import { attach, createErrorHandler, usePagination } from '@openmrs/esm-framework';
-import { formatDate } from './biometric.helper';
-import { useNotesContext } from './notes.context';
+import { attach, createErrorHandler, getStartedVisit, usePagination, VisitItem } from '@openmrs/esm-framework';
+import Add16 from '@carbon/icons-react/es/add/16';
+import Button from 'carbon-components-react/es/components/Button';
+import DataTableSkeleton from 'carbon-components-react/es/components/DataTableSkeleton';
+import DataTable, {
+  Table,
+  TableCell,
+  TableContainer,
+  TableBody,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from 'carbon-components-react/es/components/DataTable';
 import { getEncounterObservableRESTAPI, PatientNote } from './encounter.resource';
+import { formatNotesDate } from './notes-helper';
+const notesToShowCount = 5;
 
 interface NotesDetailedSummaryProps {
   showAddNote: boolean;
+  patientUuid: string;
 }
 
-const NotesDetailedSummary: React.FC<NotesDetailedSummaryProps> = ({ showAddNote }) => {
+const NotesDetailedSummary: React.FC<NotesDetailedSummaryProps> = ({ showAddNote, patientUuid }) => {
```

Comment:
Why is the notes context dropped? The patient UUID should be received from the context.